### ISSUE TITLE
voc2007数据集在训练中问题

### ISSUE THREAD

**Issue by chairc** [bubbliiiing/yolox-pytorch#63]
我在voc2007官方数据集中使用2500训练集，2500验证集，加载预训练权重。冻结训练中的train loss和val loss稳定下降，在解冻训练后train loss和val loss升高2到3个点，在运行几个epoch后，val loss高于train loss，train loss在后续是降低，val loss反而升高。请问这是什么情况？还有请问作者在训练voc时候是如何配置的？
mosaic：开
余弦退火：开
冻结训练：50
解冻训练：100

**Comment by bubbliiiing** [comment]
…你下我的07+12
然后200代

**Comment by bubbliiiing** [comment]
你什么时候的代码

**Comment by chairc** [comment]
这次版本之前的3月2日release的那个代码，我估计是voc2007训练集太少的事情

**Comment by bubbliiiing** [comment]
那你下新的吧

**Comment by bubbliiiing** [comment]
之前的余弦退火not ok
